roadmap.md

```diff
@@ -15,23 +15,18 @@ Gazebo classic to Ignition. Take a look at the
 [feature comparison](/docs/citadel/comparison) page for a list of
 differences between classic and Ignition.
 
-## 2020 Q3(Jul - Sep)
+## 2020 Q3(Jul - Sep) - done
```

Comment:
How about "finished" instead of "done"? Done implies that there's nothing left to do.